pr_check.sh

```diff
@@ -21,17 +21,28 @@ source $CICD_ROOT/build.sh
 # Deploy the new image to an ephemeral environment
```

Comment:
@tahmidefaz We need to build the **playbook-dispatcher-connect** image before deploying the components to the ephemeral env and we also need to add a new env var that is going to be used by the `deploy_ephemeral_env.sh ` script.

EXTRA_DEPLOY_ARGS="--set-image-tag ${IMAGE_CONNECT}=${IMAGE_TAG}"